main.js

```diff
@@ -231,13 +230,15 @@ app = {
 
 	go: () => {
 		const wordLink = $(".tool-type-img").prop("src");
-		if (wordLink !== "http://s0urce.io/client/img/words/template.png") {
+		if (wordLink !== "http://www.s0urce.io/client/img/words/template.png") {
```

Comment:
cause i use the debug log for seen how the bot understud the link and she seen like this so make what the bot have say to me